extensions/superboogav2/chat_handler.py

```diff
@@ -51,17 +51,17 @@ def _format_single_exchange(name, text):
 
 
 def _get_names(state: dict):
-    if _is_instruct_mode(state):
-        user_name = state['name1_instruct']
-        bot_name = state['name2_instruct']
-    else:
-        user_name = state['name1']
-        bot_name = state['name2']
-
-    if not user_name:
-        user_name = 'User'
-    if not bot_name:
-        bot_name = 'Assistant'
+    user_name = 'User'
+    bot_name = 'Assistant'
+    try:
+        if _is_instruct_mode(state):
+            user_name = state['name1_instruct']
```

Comment:
The name1_instruct and name2_instruct fields do not exist anymore. It would be better to remove that part and use the updated chat functions instead.

See this function: https://github.com/oobabooga/text-generation-webui/blob/19d13743a6c436474f80643c26eb1fb35e5446f5/extensions/openai/completions.py#L192

Reply:
Ah, that makes life easier then. I've pulled that bit out, and put in a substitute based on the linked function.